internal/log/console_windows.go

```diff
@@ -0,0 +1,35 @@
+package log
+
+import (
+	"golang.org/x/sys/windows"
+	"syscall"
+	"log"
+)
+
+// EnableColoredConsoleOutput sets flag that enables VT escape sequences
+// in windows console. See link for Win API documentation.
+//
+// https://docs.microsoft.com/en-us/windows/console/setconsolemode
+//
+func EnableColoredConsoleOutput() {
+	var console_handle syscall.Handle
+	var err error
+
+	console_handle, err = syscall.Open("CONOUT$", syscall.O_RDWR, 0)
+	if err != nil {
+		log.Println("Error getting console handle:", err)
```

Comment:
Better use our `internal/log` package with `log.Errorf` in that place (also for other other two cases).